BioForm/profiles/static/styles/blog.css

```
@@ -0,0 +1,87 @@
+.user {
+    font-weight: bold;
+    font-family:Georgia, 'Times New Roman', Times, serif;
+}
+
+.heading {
+    clear: both;
+    text-align: center;
+    color: lightgoldenrodyellow;
+}
+
+.create_text {
+    text-decoration: none;
+    color: lightyellow;
+    margin: 45px;
+    margin-right: 20px;
+    background-color: #f7ba5b; /* Green */
+    border: none;
+    padding: 15px 32px;
+    text-align: center;
+    font-size: 16px;
+    border-radius:25px;
+}
+
+.created{
+    margin-bottom: 34px;
+}
+
+.messages {
+    margin: 30px;
+    margin-left: 58px;
+    color: darkred;
+    font-weight: bold;
+}
+
+.record {
+    margin-top: 60px;
+    margin-left: 80px;
+    width: calc(90%);
```

You don't need `calc` here, it's usually used when you want to do some calculation based on some dynamic value e.g `calc(90% - 10px)`.
```suggestion
    width: 90%;
```

---

I did text-overflow: ellipsis and the only way that works is if the width is calculated in absolute value. The calc(90%) basically converts the width into px according to device so that’s why I did that. If i remove it, the ellipses will stop working again